packages/ocapn/test/_table.js

```diff
@@ -0,0 +1,523 @@
+const sym = s => `${s.length}'${s}`;
+const str = s => `${s.length}"${s}`;
```

Comment:
Consider making the expected behavior for `>0xff` words explicit, either forbidding or tolerating correctly. That would look like:

```suggestion
const str = s => {
  const b = new TextEncoder().encode(s);
  return `${b.byteLength}"${String.fromCharCode(...b)}`;
};
```

Same for sym/sel